Automattic/jetpack
# Provide a fix for WPA click tracking in Agencies card

Fixes #25041

#### Changes proposed in this Pull Request:
* Fix link tracking in Agencies card to be triggered on click, not render

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:
* Start the Jetpack Plugin locally on Docker, or however you go about testing the plugin locally
* Apply any Jetpack Plan to your local wordpress site if you don't already have one active
* Go to the Jetpack Dashboard at `/wp-admin/admin.php?page=jetpack#/dashboard`
* Make sure you are on the `At a Glance` tab and scroll down until you see the new Agency module
![image](https://user-images.githubusercontent.com/8419292/202769053-50703a80-61b7-49c5-956b-812b77478f74.png)
* Ensure that `jetpack_wpa_click` hasn't been fired in your session yet (e.g. with Tracks Vigilante)
* Click the "Learn more" button
* Validate that the `jetpack_wpa_click` event has been fired with `target: 'learn-more-click'`
